### PR TITLE
changed .mergify.yml to use queue instead of merge actions

### PR DESCRIPTION
**Pull request by nxei**
## Motivation/summary

It changes merge action to queue in .mergify.yml

The configuration uses the deprecated strict mode of the merge action.
A brownout is planned for the whole December 6th, 2021 day.
This option will be removed on January 10th, 2022.
For more information: https://blog.mergify.io/strict-mode-deprecation/

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -75,13 +79,13 @@ pull_request_rules:
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
   - name: Automatic squash and merge with success checks and the file docker-compose.yml is modified.
     conditions:
-      - check-success=apm-ci/pr-merge
       - label=automation
       - files=docker-compose.yml
+      - check-success=apm-ci/pr-merge
```
Comment (v1v) [inline .mergify.yml:84]: This change is not required, can you put it back in the above line?

File: .mergify.yml [hunk 2]
```diff
@@ -95,14 +99,14 @@ pull_request_rules:
       delete_head_branch:
   - name: automatic merge when CI passes for the make update-beats
     conditions:
-      - check-success=apm-ci/pr-merge
       - label=automation
       - base~=^update-beats
       - files~=^(go.mod|go.sum|NOTICE.txt)
+      - check-success=apm-ci/pr-merge
```
Comment (v1v) [inline .mergify.yml:105]: ditto

Comment (nxei) [reply .mergify.yml]: moved - check-success=beats-ci/pr-merge to the begin of the list

File: .mergify.yml (hunk 1 above)
Comment (nxei) [reply .mergify.yml]: moved - check-success=beats-ci/pr-merge to the begin of the list

**Review by v1v — APPROVED** [review]
LGTM

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**